requires-install.txt

```diff
@@ -5,3 +5,5 @@ dash_core_components==2.0.0
 dash_table==5.0.0
 importlib-metadata==4.8.3;python_version<"3.7"
 contextvars==2.4;python_version<"3.7"
+typing_extensions==4.1.1;python_version<"3.7"
+typing_extensions>=4.4.0;python_version>"3.6"
```

Comment:
```suggestion
typing_extensions>=4.4.0;python_version>="3.7"
```
Just to be more obviously symmetric with the `<"3.7"`

I wonder though, if 4.1.1 is sufficient would we be better off just collapsing these into one line? If someone on 3.7+ has a version >=4.1.1,<4.4.0 pinned for some reason is that a problem?
```
typing_extensions>=4.1.1
```

Reply:
4.1.1 has fix for imports on 3.7, it's the last version that works with 3.6. I don't see anything we actually uses in the later version so we can try a single version >=4.1.1